### PR TITLE
docs(): add variable descriptions

### PR DESCRIPTION
**Pull request by tkrupers**
#### Short description of what this resolves:
Short description of sass variables for action sheet, alert, card, checkbox and content

#### Changes proposed in this pull request:

- Description of sass variables

**Ionic Version**: 1.x / 2.x

**Fixes**: #8785

### CODE REVIEW

File: src/components/card/card.md.scss [hunk 1]
```diff
@@ -3,36 +3,81 @@
...
+/// @prop - Padding bottom of the card
 $card-md-padding-bottom:              13px !default;
+
+/// @prop - Padding left of the card
 $card-md-padding-left:                16px !default;
 
+/// @prop - Padding top of the media on the card
 $card-md-padding-media-top:           10px !default;
+
+/// @prop - Padding top of the media on the card
```
Comment (brandyscarney) [inline src/components/card/card.md.scss]: Can this be changed to "Padding bottom"?

File: src/components/card/card.wp.scss [hunk 2]
```diff
@@ -3,37 +3,82 @@
...
+/// @prop - Padding bottom of the card
 $card-wp-padding-bottom:              13px !default;
+
+/// @prop - Padding left of the card
 $card-wp-padding-left:                16px !default;
 
+/// @prop - Padding top of the media on the card
 $card-wp-padding-media-top:           10px !default;
+
+/// @prop - Padding top of the media on the card
```
Comment (brandyscarney) [inline src/components/card/card.wp.scss]: Same here, Padding bottom :)

File: src/components/alert/alert.ios.scss [hunk 3]
```diff
@@ -3,90 +3,208 @@
...
 $alert-ios-button-border-radius:                0 !default;
 
+/// @prop - Font weight of the main text on the alert button
 $alert-ios-button-main-font-weight:             bold !default;
 
+/// @prop - Border top of the alert list
 $alert-ios-list-border-top:                     $alert-ios-button-border-width $alert-ios-button-border-style $alert-ios-button-border-color !default;
 
+/// @prop - Padding on the label for the radio alert button
 $alert-ios-radio-label-padding:                 13px !default;
```
Comment (brandyscarney) [inline src/components/alert/alert.ios.scss]: Starting here to the end of the file the word "button" is added to the radio and checkbox descriptions. Could you remove this or change it to "input"? This is also in the other mode files. :)

**Comment by tkrupers** [comment]
Thanks for your quick reply! I've updated the PR.

**Comment by brandyscarney** [comment]
Looks great! Thank you. :)

**Pull request merged**